games/src/main/java/brocoeur/example/games/service/blackjack/BlackJackService.java

```diff
@@ -0,0 +1,35 @@
+package brocoeur.example.games.service.blackjack;
+
+import brocoeur.example.common.GamePlay;
+import brocoeur.example.common.GameStrategy;
+import brocoeur.example.games.service.GameRound;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
- Why à service is using the component annotation?

Reply:
Fixed thanks for noticing